src/DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -76,6 +77,11 @@ function PreFirstEntryMapFixes()
             k.Description = "Control Room Door Key";
             if(dxr.flags.settings.keysrando > 0)
                 GlowUp(k);
+
+            //Make the ambrosia worth more so that there's more incentive to actually do your job?
+            foreach AllActors(class'#var(prefix)SkillAwardTrigger', sat, 'AmbrosiaTagged'){
+                sat.skillPointsAdded=300; //Default is 150
```

Comment:
also you get 500 credits, so I guess that's something lol, this isn't hard to do so it seems pretty worth it